internal/db/query.go

```diff
@@ -176,6 +187,10 @@ func loadChunk(ctx context.Context, tx pgx.Tx, lookup *ContentLookup, dbObjects
 type ObjectStream func() (*pb.Object, error)
 
 func GetObjects(ctx context.Context, tx pgx.Tx, lookup *ContentLookup, packManager *PackManager, project int64, vrange VersionRange, objectQuery *pb.ObjectQuery) (ObjectStream, error) {
+	return GetObjectsSizeLimited(ctx, tx, lookup, packManager, project, vrange, objectQuery, -1)
```

Comment:
same double function comment, I think it'd be fine to just have one that takes an optional / defaulted arg